keyboards/nixkeyboards/day_off/info.json

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
VIA JSON is not info.json. VIA JSON goes in the VIA repo. Please read the docs.

Reply:
info.json is then just the KLA?

Reply:
No. KLE is yet another format.
https://docs.qmk.fm/#/reference_info_json?id=infojson
Also have a look at other keyboards to get a sense of what this file should look like.